libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -112,7 +112,7 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @Param: FS_LOW_ACT
     // @DisplayName: Low battery failsafe action
     // @Description: What action the vehicle should perform if it hits a low battery failsafe
-    // @Values{Plane}: 0:None,1:RTL,2:Land,3:Terminate,4:QLand,6:Loiter to QLand
+    // @Values{Plane}: 0:None,1:RTL,2:Land,3:Terminate,4:QLand,5:Parachute release,6:Loiter to QLand,7:AUTOLAND or RTL
```

Comment:
Also need to update `FS_CRT_ACT` Parachute is enumerated there already.

Reply:
did not feel the real need to add this to crt fs....

Reply:
The value will still work whether or not you add it to the description.

Reply:
done